test/state/state.hpp

```diff
@@ -103,11 +109,23 @@ struct Log
 
 struct TransactionReceipt
 {
+    bool status = false;
```

Comment:
```suggestion
    evmc_status_code status = EVMC_INTERNAL_ERROR;
```
I think we can keep the full status code and convert to bool in RLP encoding.